src/EventSubscriber/AddContentLanguageHeaderSubscriber.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\HttpCache\EventSubscriber;
+
+use eZ\Publish\API\Repository\Values\Content\Content;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\KernelEvents;
+use Symfony\Component\HttpKernel\Event\ResponseEvent;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+
+final class AddContentLanguageHeaderSubscriber implements EventSubscriberInterface
+{
+    public const CONTENT_LANGUAGE_HEADER = 'x-lang';
+    private const CONTENT_VIEW_ROUTES = ['_ez_content_view', 'ez_urlalias'];
```

Comment:
Would be nice if this could be extendable since the class is `final` and the const is `private`. There is no possibility to override this setting which means someone would have to duplicate this subscriber to add support for their own routes.

Reply:
Side comment: for 4.0.x-dev you will need to rebrand those routes.